scripts/run-operator.sh

```diff
@@ -15,9 +15,13 @@ then
   minikube start
 fi
 
-echo "building operator image in minikube"
+GIT_HASH="$(git rev-parse HEAD)"
+TMP_INSTALL_DIR="$(mktemp -d)"
+trap 'rm -rf -- "$TMP_INSTALL_DIR"' EXIT
+
+echo "building operator image in minikube for commit ${GIT_HASH}"
```

Comment:
One thing I noticed when using this script is that it's hard to spot the script output amongst the output of the various other commands it runs. I wondered whether using some consistent prefix might help.